.github/workflows/ci.yml

```diff
@@ -69,9 +69,9 @@ on:
       AZURE_CLIENT_SECRET:
         required: false
         description: "Azure clients secret, needs to be rotated before 2025-12-21 (see the pulumi-test user in Azure portal)"
-      GCP_SERVICE_ACCOUNT:
```

Comment:
Shouldn't we still have the GCP_SERVICE secret as well?

Reply:
Yup, this seems like a mismerge, thanks for catching that!